include/z64camera.h

```diff
@@ -0,0 +1,389 @@
+    /* 0x08 */ CAM_SET_DOORC, // Generic room door transitions, camera moves and follows player as the door is open and closed
+    /* 0x09 */ CAM_SET_DEMO0, // Unknown, possibly related to treasure chest game as goron?
+    /* 0x0A */ CAM_SET_FREE0, // Free Camera, manual control is given, no auto-updating eye or at
+    /* 0x0B */ CAM_SET_BIRDS_EYE_VIEW_0, // Appears unused. Camera is a top-down view "FUKAN0"
+    /* 0x0C */ CAM_SET_NORMAL1, // Generic camera 1, used in various places
+    /* 0x0D */ CAM_SET_NANAME, // Unknown, slanted or tilted. Behaves identical to Normal0 except with added roll
+    /* 0x0E */ CAM_SET_CIRCLE0, // Used in Curiosity Shop, Pirates Fortress, Mayor's Residence
+    /* 0x0F */ CAM_SET_FIXED0, // Used in Sakon's Hideout puzzle rooms, milk bar stage
+    /* 0x10 */ CAM_SET_SPIRAL_DOOR, // Exiting a Spiral Staircase "SPIRAL"
+    /* 0x11 */ CAM_SET_DUNGEON0, // Generic dungeon camera 0, used in various places
+    /* 0x12 */ CAM_SET_ITEM0, // Getting an item and holding it above Player's head (from small chest, freestanding, npc, ...)
+    /* 0x13 */ CAM_SET_ITEM1, // Looking at player while playing the ocarina
+    /* 0x14 */ CAM_SET_ITEM2, // Bottles: chugging, releasing fairy, dropping fish
+    /* 0x15 */ CAM_SET_ITEM3, // Bottles: catching fish or bugs, showing an item
+    /* 0x16 */ CAM_SET_NAVI, // Song of Soaring, variations of playing Song of Time "NAVI"
+    /* 0x17 */ CAM_SET_WARP_PAD_MOON, // Warp circles from Goron Trial on the moon "WARP0"
+    /* 0x18 */ CAM_SET_DEATH, // Player death animation when health goes to 0 "DEATH"
+    /* 0x19 */ CAM_SET_REBIRTH, // Unknown set with camDataId = -9 (it's not being revived by a fairy)
+    /* 0x1A */ CAM_SET_LONG_CHEST_OPENING, // Long cutscene when opening a big chest with a major item "TREASURE"
+    /* 0x1B */ CAM_SET_MASK_TRANSFORMATION, // Putting on a transformation mask "TRANSFORM"
+    /* 0x1C */ CAM_SET_ATTENTION, // Unknown, set with camDataId = -15
+    /* 0x1D */ CAM_SET_WARP_PAD_ENTRANCE, // Warp pad from start of a dungeon to the boss-room "WARP1"
+    /* 0x1E */ CAM_SET_DUNGEON1, // Generic dungeon camera 1, used in various places
+    /* 0x1F */ CAM_SET_FIXED1, // Fixes camera in place, used in various places eg. entering Stock Pot Inn, hiting a switch, giving witch a red potion, shop browsing
+    /* 0x20 */ CAM_SET_FIXED2, // Used in Pinnacle Rock after defeating Sea Monsters, and by Tatl in Fortress
+    /* 0x21 */ CAM_SET_MAZE, // Unused. Set to use Camera_Parallel2(), which is only Camera_Noop()
+    /* 0x22 */ CAM_SET_REMOTEBOMB, // Unused. Set to use Camera_Parallel2(), which is only Camera_Noop(). But also related to Play_ChangeCameraSetting?
+    /* 0x23 */ CAM_SET_CIRCLE1, // Unknown
+    /* 0x24 */ CAM_SET_CIRCLE2, // Looking at far-away NPCs eg. Garo in Road to Ikana, Hungry Goron, Tingle
+    /* 0x25 */ CAM_SET_CIRCLE3, // Used in curiosity shop, goron racetrack, final room in Sakon's hideout, other places
```

Comment:
Yeah, these comments were made months ago when very few actors were decompiled, and were just comments on observations in game. As the decompilation reaches 100%, the enum usage in these actors will speak for themselves and these names will be more about how they pivot instead of how they're used